packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/index.ts

```diff
@@ -101,6 +101,7 @@ class IssuePrepaidCardWorkflow extends Workflow {
           },
         }),
         new NetworkAwareWorkflowCard({
+          cardName: 'HUB_AUTH',
```

Comment:
I added these names to all cards so that we are able to reference the last completed card. I was thinking about persisting the `componentName` but I figured there will be problems in case someone changes the component name.